runtime-infra-new/pn-ipc-dev-cfg.json

```diff
@@ -4,6 +4,7 @@
     "ApiCertificateArn": "arn:aws:acm:eu-central-1:558518206506:certificate/30124909-4782-4e4e-9802-2efb1d200a02",
     "WebApiDnsName": "webapi.dev.pn.pagopa.it",
     "WebApiCertificateArn": "arn:aws:acm:eu-central-1:558518206506:certificate/e65813bb-4ed9-4750-b9c1-f2ef815efe63",
-    "HostedZoneId": "Z0574674KM4X77EEGT5U"
+    "HostedZoneId": "Z0574674KM4X77EEGT5U",
```

Comment:
Questi Id arrivano dagli script di creazione delle Zone DNS riusciamo a fare in modo che siano recuperati o compilati da uno script? Altrimenti quando dobbiamo passare su poste dobbiamo ricordarci di cambiarli tutti. Vero?

Reply:
Si, ma ci succederà due o tre volte nel tempo di vita di PN.
E i file da cambiare sono tutti in questa directory.